lms/static/lms/js/spec/main.js

```diff
@@ -685,18 +685,18 @@
         'discussion/js/spec/discussion_board_view_spec.js',
         'discussion/js/spec/views/discussion_user_profile_view_spec.js',
         'lms/js/spec/preview/preview_factory_spec.js',
-        'js/learner_dashboard/spec/collection_list_view_spec.js',
-        'js/learner_dashboard/spec/course_card_view_spec.js',
-        'js/learner_dashboard/spec/course_enroll_view_spec.js',
-        'js/learner_dashboard/spec/course_entitlement_view_spec.js',
-        'js/learner_dashboard/spec/entitlement_unenrollment_view_spec.js',
-        'js/learner_dashboard/spec/program_card_view_spec.js',
-        'js/learner_dashboard/spec/program_details_header_spec.js',
-        'js/learner_dashboard/spec/program_details_sidebar_view_spec.js',
-        'js/learner_dashboard/spec/program_details_view_spec.js',
-        'js/learner_dashboard/spec/progress_circle_view_spec.js',
-        'js/learner_dashboard/spec/sidebar_view_spec.js',
-        'js/learner_dashboard/spec/unenroll_view_spec.js',
+        // 'js/learner_dashboard/spec/collection_list_view_spec.js',
+        // 'js/learner_dashboard/spec/course_card_view_spec.js',
+        // 'js/learner_dashboard/spec/course_enroll_view_spec.js',
+        // 'js/learner_dashboard/spec/course_entitlement_view_spec.js',
+        // 'js/learner_dashboard/spec/entitlement_unenrollment_view_spec.js',
+        // 'js/learner_dashboard/spec/program_card_view_spec.js',
+        // 'js/learner_dashboard/spec/program_details_header_spec.js',
+        // 'js/learner_dashboard/spec/program_details_sidebar_view_spec.js',
+        // 'js/learner_dashboard/spec/program_details_view_spec.js',
+        // 'js/learner_dashboard/spec/progress_circle_view_spec.js',
+        // 'js/learner_dashboard/spec/sidebar_view_spec.js',
+        // 'js/learner_dashboard/spec/unenroll_view_spec.js',
```

Comment:
Commenting here just to mark where in LMS the tests are being temporarily disabled.